test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        //empty list
+        assertNotNull();
+        //list with null
+        assertAnyNull(1,null);
+        //list with no null
+        assertNotNull(1,2,3);
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertAnyNull(Object... objects) { assertTrue(Utils.isAnyNull(objects));}
```

Comment:
👍 It's good to see you following the style of the nearby code i.e., extract utility methods asserting true and false cases. 🤔 But it seems, in this case, there is not much value is extracting those as separate functions as it doesn't make the code easier/understand to read or shorter.